.github/dependabot.yml

```
@@ -8,3 +8,11 @@ updates:
       interval: "weekly"
     reviewers:
       - "gardenlinux/garden-linux-maintainers"
+
+  - package-ecosystem: "pip"
+    directory: "/"
```

```suggestion
    directory: "/tests"
```

To my understanding this should point to the directory where our pipfile is located

---

Do we need to limit dependabot to check only Pipfile in tests? Why?

---

Checking if dependabot scans recursively or needs explicit paths

---

Yep - you were right

---

fyi, dependabot docs describe this: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#directory